services/web/server/tests/integration/02/test_rabbit.py

```diff
@@ -199,8 +201,23 @@ def client(
 
 
 @pytest.fixture
-def client_session_id() -> UUIDStr:
-    return str(uuid4())
+def user_id(faker: Faker) -> int:

```

Comment:
does the faker library already provide this fixture?

Reply:
https://faker.readthedocs.io/en/master/pytest-fixtures.html